packages/react-devtools-shared/src/__tests__/__snapshots__/profilingCache-test.js.snap

```diff
@@ -1520,19 +1510,7 @@ Object {
   },
   "interactionCommits": Map {},
   "interactions": Map {},
-  "operations": Array [
-    Array [
-      1,
-      6,
-      0,
-      2,
-      4,
-      9,
-      8,
-      7,
-      6,
```

Comment:
Just to better annotate this change:
```sh
  1,  # rendered ID
  6,  # root ID
  0,  # string table isze
  2,  # TREE_OPERATION_REMOVE operation
  4,  # number of Fibers removed
  9,  # ids...
  8,  
  7,  
  6,
```

As explained in the PR description, it was expected.